tekton/src/pipelines/gitops/deprovision-mas-instance.yml.j2

```diff
@@ -192,6 +196,46 @@ spec:
       taskRef:
         kind: Task
         name: gitops-process-mongo-user
+    - name: gitops-deprovsion-db2u
+      runAfter:
+        - gitops-process-mongo-user
```

Comment:
Does Db2 need EFS? If so then we probably need to run it before the deprovison-efs which is currently going to happen at the same time. I believe we can remove db2 at the same time as mongo user, so just change this gitops-deprovsion-db2u to runAfter: gitops-deprovision-suite